config/locales/pl.yml

```
@@ -757,6 +758,16 @@ pl:
     older: Starsze
     prev: Poprzednia
     truncate: "&hellip;"
+  polls:
+    errors:
+      already_voted: Już zagłosowałeś(-aś) w tym głosowanie
+      duplicate_options: zawiera powtarzające się opcje
+      duration_too_long: jest zbyt odległa
+      duration_too_short: jest zbyt bliska
+      expired: To głosowanie już zakończyło się
+      over_character_limit: nie mogą składać się z więcej niż %{MAX} znaków
```

```suggestion
      over_character_limit: nie mogą składać się z więcej niż %{max} znaków
```
%{MAX} was recently changed to %{max}, please change this line.